### PR TITLE
fix: SEO: fix broken links

### PR DESCRIPTION
**Pull request by ch3ric**
* Fix broken links
* Replace links to codelabs by blog links
* Use `{BASE_URL}` for all internal links in order to avoid nofollow attributes + add doc about this

### CODE REVIEW

**Comment by ch3ric** [comment]
Careful: when this PR is merged, we need to ask @Cindyvlv to enable Axept.io via GTM at the same time.

File: _articles/fr/2020-08-26-a-la-decouverte-de-mercure.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ Bien évidemment, ce protocole est beaucoup plus complet qu'il n'y parait, et il
 
 ![GIF Keep your secrets]({BASE_URL}/imgs/articles/2020-08-26-a-la-decouverte-de-mercure/keep_your_secrets.gif)
 
-Cependant, un codelabs est prévu très bientôt pour approfondir tout cela, et découvrir plein d'autres choses, notamment comment sécuriser les connexions, utiliser Mercure dans une application Symfony... Le tout sur la plateforme [Eleven's Codelabs](https://codelabs.eleven-labs.com/).
+Cependant, un codelabs est prévu très bientôt pour approfondir tout cela, et découvrir plein d'autres choses, notamment comment sécuriser les connexions, utiliser Mercure dans une application Symfony... Le tout sur la plateforme [Eleven's Codelabs](/fr/creez-un-chat-avec-symfony-et-mercure/).
```
Comment (Shad-Z) [inline _articles/fr/2020-08-26-a-la-decouverte-de-mercure.md:168]: i see some links for codelabs with `{BASE_URL}` like https://github.com/eleven-labs/blog.eleven-labs.com/pull/1045/files#diff-626ff138ff10ddebefea919de55180f16c1ed6986a32b2f2bed69a3828e99d4eR29 and some without which one is good ?

**Review by lepiaf — APPROVED** [review]

File: _articles/fr/2020-08-26-a-la-decouverte-de-mercure.md (hunk 1 above)
Comment (lepiaf) [reply _articles/fr/2020-08-26-a-la-decouverte-de-mercure.md:168]: it works fine http://dev.blog.eleven-labs.com.s3-website.eu-west-2.amazonaws.com/fix/broken-links/fr/creez-un-chat-avec-symfony-et-mercure/

**Review by snroki — APPROVED** [review]

**Review by Pouzor — APPROVED** [review]

File: _articles/fr/2020-08-26-a-la-decouverte-de-mercure.md (hunk 1 above)
Comment (ch3ric) [reply _articles/fr/2020-08-26-a-la-decouverte-de-mercure.md:168]: Both works but better use always BASE_URL in order for all links to work in all envs including dev. I'm gonna fix the ones I've missed

**Review by Shad-Z — APPROVED** [review]

**Review by lepiaf — APPROVED** [review]

**Pull request merged**